src/backend/drm/surface/gbm.rs

```diff
@@ -56,6 +44,7 @@ where
     pub fn new(
         drm: DrmSurface,
         mut allocator: A,
+        color_formats: &[Fourcc],
```

Comment:
Maybe this could benefit from a comment in the functions docs describing that the formats are
tested in the order they appear.